config/.env

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    secret: process.env.SECRET_KEY || "s3cr3tH",
+    linkedin: {
+        consumerKey: '78rpcb3ijnju51',
```

Comment:
We can not use the key here exposed on the repo, we need to replace by `process.env.LINKEDIN_CONSUMER_KEY` and the same for consumer secret

Reply:
Sorry. I have had add  the .env on .gitignore but not works as well. I already removed from stage

Reply:
@felipehfs , I think you have to keep on `.gitignore`, and this file .env file will be a javascript file, and then you have a local env ignored in gitignore with the secrets values in the following format:

```
LINKEDIN_CONSUMER_KEY=number
```
Then this file will be a env and a javascript file will reference by `process.env.NAME`

You got it now? If you check the Gitpay source you will see how we setup this

Reply:
Alright. Can you send me the source file?